arches/app/media/js/views/components/search/map-filter.js

```diff
@@ -274,6 +275,35 @@ define([
                 }
             }, this);
 
+            this.filterByFeatureGeom = function(feature, resourceid=null) {
+                if (feature.geometry.type == 'Point') { self.buffer(100); }
```

Comment:
Do we have access to the buffer value in the filter settings?

Reply:
we have access to the buffer in this context given how it's defined [here](https://github.com/archesproject/arches/blob/dev/7.6.x/arches/app/media/js/viewmodels/map-filter.js#L14) in the map-filter viewmodel

Reply:
for good measure I changed the above line to set a buffer of 25 and only if the existing value for `this.buffer() == 0`